base/applications/rapps/include/rosui.h

```diff
@@ -96,9 +96,13 @@ class CPointerArray
 
     BOOL RemoveAt(INT i)
     {
-        T* ptr = (T*) DPA_GetPtr(m_hDpa, i);
-        OnRemoveItem(ptr);
-        return DPA_DeletePtr(m_hDpa, i);
+        T* ptr = DPA_DeletePtr(m_hDpa, i);
```

Comment:
`DPA_DeletePtr` returns PVOID so this needs `static_cast<T*>`. How does this compile?

Reply:
it's still better than what it was before ;-)

A quick check on godbolt indicates that one must pass -fpermissive flag to G++ to have this compile.